docs/can-guides/commitment/recipes/file-navigator/file-navigator-advanced-7.js

```diff
@@ -44,55 +46,70 @@ const makeEntities = function(parentId, depth) {
 const entities = makeEntities("0", 0);
 
 // Add them to a client-like DB store
-const entitiesStore = can.fixture.store(entities);
+const entitiesStore = fixture.store(entities);
 
 // Trap requests to /api/entities to read items from the entities store.
-can.fixture("/api/entities", entitiesStore);
+fixture("/api/entities", entitiesStore);
 
 // Make requests to /api/entities take 1 second
-can.fixture.delay = 1000;
+fixture.delay = 1000;
 
-const Entity = can.DefineMap.extend({
-  id: "string",
+const Entity = DefineMap.extend({
+  id: {type: "string", identity: true},
   name: "string",
   parentId: "string",
   hasChildren: "boolean",
   type: "string"
 });
 
-Entity.List = can.DefineList.extend({});
-
-can.connect.baseMap({
+Entity.connection = restModel({
   Map: Entity,
   url: "/api/entities"
 });
 
-const folder = new Entity({
-  id: "0",
-  name: "ROOT/",
-  hasChildren: true,
-  type: "folder"
-});
-
-const FolderVM = can.DefineMap.extend({
-  folder: Entity,
-  entitiesPromise: {
-    default: function() {
-      return Entity.getList({parentId: this.folder.id});
+Component.extend({
+  tag: "a-folder",
+  view: `
+    <span on:click="toggleOpen()">{{ this.folder.name }}</span>
```

Comment:
I think this should be `this.toggleOpen()`

…or actually, can we replace this with just `this.isOpen = !this.isOpen` inside the click handler? These are the docs you could link to: https://canjs.com/doc/can-stache-bindings.event.html#on_VIEW_MODEL_OR_DOM_EVENT__KEY_VALUE_

Reply:
This is not working, maybe a bug, because `this.isOpen = true` works

Reply:
Hm, ok, would you mind filing an issue please?